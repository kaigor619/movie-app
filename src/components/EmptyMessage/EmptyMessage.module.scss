.emptyMessage {
  flex: 1;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;

  & > p {
    color: rgba(var(--text-primary-rgb), 0.5);
    font-size: 1.2rem;
    margin: 1rem 0 0 0;
  }
}
