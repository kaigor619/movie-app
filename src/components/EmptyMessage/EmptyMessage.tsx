import styles from "./EmptyMessage.module.scss";
import noFoundImg from "./money.png";

export const EmptyMessage: React.FC = () => {
  return (
    <div className={styles.emptyMessage}>
      <img src={noFoundImg} alt="" />
      <p>Nothing was found</p>
    </div>
  );
};
