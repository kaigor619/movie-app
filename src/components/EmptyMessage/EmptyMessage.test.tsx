import { render, screen } from "@testing-library/react";
import { EmptyMessage } from "./EmptyMessage";

describe("EmptyMessage component", () => {
  it("displays image and corresponding text", () => {
    const { asFragment } = render(<EmptyMessage />);

    expect(screen.getByRole("img")).toBeInTheDocument();
    expect(screen.getByText(/nothing was found/i)).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
