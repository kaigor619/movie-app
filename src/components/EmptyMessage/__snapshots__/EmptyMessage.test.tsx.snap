// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyMessage component displays image and corresponding text 1`] = `
<DocumentFragment>
  <div
    class="emptyMessage"
  >
    <img
      alt=""
      src="money.png"
    />
    <p>
      Nothing was found
    </p>
  </div>
</DocumentFragment>
`;
